packages/manager/modules/pci/src/projects/new/payment/index.html

```diff
@@ -74,36 +74,55 @@ <h1 data-translate="pci_project_new_payment_title"></h1>
                         data-type="submit"
                         data-variant-nav="next"
                         data-disabled="$ctrl.globalLoading.finalize"
-                        data-ng-if="!$ctrl.isCheckingPaymentMethod"
+                        data-ng-if="!$ctrl.needToCheckCustomerInformations || !$ctrl.isCheckingPaymentMethod"
                     >
                         <span
                             data-translate="{{ 'pci_project_new_payment_btn_continue_' + $ctrl.model.paymentMethod.paymentType.toLowerCase() }}"
                         ></span>
                     </oui-button>
                 </ovh-payment-method-integration>
 
-                <oui-button
-                    data-variant="primary"
-                    data-type="submit"
-                    data-variant-nav="next"
-                    data-disabled="!$ctrl.model.credit"
-                    data-ng-if="$ctrl.model.paymentMethod.paymentType === 'CREDIT'"
+                <!--Go to billing detail page-->
+                <a
+                    data-ng-if="$ctrl.needToCheckCustomerInformartions"
+                    class="oui-button oui-button_primary oui-button_icon-right text-white"
+                    data-ng-href="{{ $ctrl.orderBillingUrl }}"
+                    target="_blank"
+                    rel="noopener"
                 >
                     <span
-                        data-translate="{{ 'pci_project_new_payment_btn_continue_' + $ctrl.model.paymentMethod.paymentType.toLowerCase() }}"
+                        data-translate="pci_project_new_payment_check_anti_fraud_case_fraud_manual_review_link"
+                    ></span>
+                    <span
+                        class="oui-icon oui-icon-arrow-right"
+                        aria-hidden="true"
                     ></span>
-                </oui-button>
+                </a>
 
-                <oui-button
-                    data-variant="primary"
-                    data-type="submit"
-                    data-variant-nav="next"
-                    data-disabled="
+                <div data-ng-if="!$ctrl.needToCheckCustomerInformartions">
+                    <oui-button
+                        data-variant="primary"
+                        data-type="submit"
+                        data-variant-nav="next"
+                        data-disabled="!$ctrl.model.credit"
+                        data-ng-if="$ctrl.model.paymentMethod.paymentType === 'CREDIT'"
+                    >
+                        <span
+                            data-translate="{{ 'pci_project_new_payment_btn_continue_' + $ctrl.model.paymentMethod.paymentType.toLowerCase() }}"
+                        ></span>
+                    </oui-button>
+
+                    <oui-button
+                        data-variant="primary"
+                        data-type="submit"
+                        data-variant-nav="next"
+                        data-disabled="
```

Comment:
Looks like lot of complex logic to disable/enable this button. Can you move this to controller?

Reply:
done